specification/mediaservices/resource-manager/readme.md

```diff
@@ -26,10 +26,63 @@ These are the global settings for the MediaServices API.
 
 ```yaml
 openapi-type: arm
-tag: package-2021-06
+tag: package-2021-11
 opt-in-extensible-enums: true
 ```
 
+### Suppression
+
+```yaml
+directive:
+  - from: AssetsAndAssetFilters.json
+    reason: Patch response is 202.
+    suppress: LongRunningResponseStatusCode
+
+  - from: Accounts.json
+    reason: Patch response is 202.
+    suppress: LongRunningResponseStatusCode
+
+  - suppress: R2016
+    where: $.definitions.TrackedResource.required
+    reason: location is a required property for our patch calls
```

Comment:
this is not allowed.  If you have this validation APIs like this will fail when ARM makes the downstream call to your RP:  https://docs.microsoft.com/en-us/rest/api/resources/tags/update-at-scope

Reply:
this was a existing suppression in the readme, I just move it up, so that I don't have to keep copying it for ever new API version.

This was added because, we were still using same definition for Patch request and PUT request in the swagger.
I can add new definitions for updates, but that means we are adding 4 new definitions and also have breaking change for update feature in SDKs.

We already have some breaking features, if you insist, I can also make this change. Do you want me to reach out to SDK team to ask their opinion or is this a blocker from ARM side.

Reply:
Is this just a swagger quirk?  I only care about the API, as long as it accepts a PATCH request without location then you can ignore this comment

Reply:
Yes,, service is working correctly, it doesn't require the location anymore.. I will work with SDK team if I should create new definitions for Update, or should we keep the suppression.